define prereqs with meteor (#252)
AICoE/elyra-aidevsecops-tutorial: docs/source/pre-requisite.md

### OpenShift Environment

> If you are using Meteor, it will build a [JupyterHub][2] environment for your project from the GitHub repository URL you provide to it. Outside of Meteor, using [Operate First][1] you just need a Google account to login to [JupyterHub][2]. Using Operate First is encouraged.

Reviewer: ```suggestion
If you are using Meteor, it will build a [JupyterHub][2] environment for your project from the GitHub repository URL you provide to it. Otherwise, we recommend using [Operate First][1], where you will just need a Google account to login to [JupyterHub][2].
```

Author: changes made :)
